opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -420,7 +420,11 @@ export const roleEditContext = async (context, user, roleId, input) => {
 
 export const assignOrganizationToUser = async (context, user, userId, organizationId) => {
   if (isOnlyOrgaAdmin(user)) {
-    throw ForbiddenAccess();
+    // When user is organization admin, we make sure she is also admin of organization added
+    const isAdministratedOrga = user.administrated_organizations.some(({ id }) => id === organizationId);
+    if (!isAdministratedOrga) {
+      throw ForbiddenAccess();
+    }
```

Comment:
Could be nice to make a function for this one as we reuse it below. If we need to make some changes in the future we will have to make them only one time

Reply:
done